ogorek.go

```diff
@@ -944,6 +947,16 @@ func (d *Decoder) loadShortBinUnicode() error {
 	return nil
 }
 
+func (d *Decoder) stackGlobal() error {
+	if len(d.stack) < 2 {
+		return errStackUnderflow
+	}
+	name := d.xpop()
+	module := d.xpop()
+	d.stack = append(d.stack, Class{Module: module.(string), Name: name.(string)})
```

Comment:
This will panic if `module` or `name` are not strings. Check the implementation of the `reduce` function for how to deal with this by returning an error.